docs/api-docs/headless/overview.md

```diff
@@ -0,0 +1,65 @@
+# Introduction to Headless Commerce
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Ways to implement headless commerce](#ways-to-implement-headless-commerce)
+- [Sample integration](#sample-integration)
+- [Next steps](#next-steps)
+- [Resources](#resources)
+
+</div>
+
+This is the first article in a comprehensive developer's guide to using BigCommerce as a commerce back-end for headless storefronts. If you are not familiar with headless commerce as a concept, start by reviewing our whitepaper, [A New Era of Ecommerce: Headless Commerce](https://www.bigcommerce.com/new-era-headless-caas/), or the Help Center's [Headless Commerce Guide](https://support.bigcommerce.com/s/article/The-Headless-Approach).
+
+## Ways to implement headless commerce
+
+If you are looking to build a headless storefront powered by a BigCommerce backend without starting from scratch, BigCommerce offers multiple starter apps and pre-built solutions to choose from. For headless storefront solutions and tools, see [Headless Integrations](https://developer.bigcommerce.com/tools-resources#headless-integrations).
+
+If you need to build a custom solution, BigCommerce has APIs and [SDKs](https://developer.bigcommerce.com/tools-resources#sdks) available to support your headless architecture.
+
+### Custom solutions
+
```

Comment:
@ektravel 
We can completely remove this section and replace it with a list of links to all the sections for this guide

Reply:
@uwikaiddi Done. Please review at your convenience.